apps/studio/components/interfaces/Organization/TeamSettings/InviteMemberButton.tsx

```diff
@@ -337,6 +353,40 @@ export const InviteMemberButton = () => {
                   </FormItem_Shadcn_>
                 )}
               />
+              <InformationBox
+                defaultVisibility={false}
+                title="Single Sign-on (SSO) login option available"
+                hideCollapse={false}
+                description={
+                  <div className="space-y-4 mb-1">
+                    <p>
+                      Supabase offers single sign-on (SSO) as a login option to provide additional
+                      account security for your team. This allows company administrators to enforce
+                      the use of an identity provider when logging into Supabase.
+                    </p>
+                    <p>This is only available for organizations on teams plan or above.</p>
+                    <div className="flex items-center space-x-2">
+                      <Button asChild type="default">
+                        <Link
+                          href="https://supabase.com/docs/guides/platform/sso"
+                          target="_blank"
+                          rel="noreferrer"
+                        >
+                          How SSO works
```

Comment:
What do you think of simplifying this? Since we're already in the context of SSO, perhaps this is sufficient?
```suggestion
                          Learn more
```